strr-base-web/app/utils/date.ts

```diff
@@ -22,7 +22,12 @@ export function dateStringToDate (dateString: string): Date | null {
  * https://moment.github.io/luxon/#/formatting?id=table-of-tokens
 */
 export function dateToString (date: Date | string, format = 'y-MM-dd') {
-  return DateTime.fromJSDate(new Date(date)).toFormat(format)
+  const luxonFormat = format.replace('a', 't')
```

Comment:
No following what is happening here...

Reply:
This allows the AM PM in the datetime string to be lowercase as requested by Andy. Passing `a` in the format.